tamr_unify_client/models/project/mastering.py

```diff
@@ -110,4 +110,11 @@ def record_clusters_with_data(self):
         name = unified_dataset.name + "_dedup_clusters_with_data"
         return self.client.datasets.by_name(name)
 
-    # super.__repr__ is sufficient
+    def published_clusters_with_data(self):
+        """
+
+        :returns: The published clusters with data represented as a dataset"""
```

Comment:
Should have an `rtype` that references the concrete type of the result - see some of the other functions for examples.